docs/guide/basic-features/connection_type.md

```diff
@@ -1,33 +1,47 @@
-# Connection Pool
+# Connection Type
 
-Kitex provides short connection pool and long connection pool for different business scenarios.
+Kitex provides **Short Connection**,  **Long Connection Pool** and **Connection Multiplexing** for different business scenarios. Kitex uses Long Connection Pool by default after v0.0.2, but adjusting the Pool Config according to your need is suggested.
 
-## Short Connection Pool
+## Short Connection
 
-Without any settings, Kitex choose using short connection pool.
+Every request needs to create a connection, the performance is bad, so it is not suggested normally.
+
+Enable Short Connection：
+
+```go
+xxxCli := xxxservice.NewClient("destServiceName", client.WithShortConnection())
+```
 
 ## Long Connection Pool
 
-Initializing the client with an Option:
+Kitex enable Long Connection Pool after >= v0.0.2, default config params are as below：
 
-```go
-client.WithLongConnection(connpool.IdleConfig{
-    MaxIdlePerAddress: 10,
-    MaxIdleGlobal:     1000,
-    MaxIdleTimeout:    60 * time.Second,
-})
 ```
+connpool2.IdleConfig{
+   MaxIdlePerAddress: 10,
+   MaxIdleGlobal:     100,
+   MaxIdleTimeout:    time.Minute,
+}
+```
+
+Adjusting the Pool Config according to your need is suggested, config as below:
+
+```
+xxxCli := xxxservice.NewClient("destServiceName", client.WithLongConnection(connpool.IdleConfig{10, 1000, time.Minute}))
+```
+
+Parameter description:
 
 - `MaxIdlePerAddress`: the maximum number of idle connections per downstream instance
 - `MaxIdleGlobal`: the global maximum number of idle connections
-- `MaxIdleTimeout`: the idle duration of the connection, connection that exceed this duration would be closed (minimum value is 3s, default value is 30s)
+- `MaxIdleTimeout`: the idle duration of the connection, a connection that exceeds this duration would be closed (minimum value is 3s, the default value is 30s)
 
-## Internal Implementation
+### Internal Implementation
 
 Each downstream address corresponds to a connection pool, the connection pool is a ring composed of connections, and the size of the ring is `MaxIdlePerAddress`.
 
 When getting a connection of downstream address, proceed as follows:
-1. Try to fetch a connection from the ring, if fetching failed (no idle connections remained), then try to  establish a new connection. In other words, the number of connections may exceed `MaxIdlePerAddress`
+1. Try to fetch a connection from the ring, if fetching failed (no idle connections remained), then try to establish a new connection. In other words, the number of connections may exceed `MaxIdlePerAddress`
 2. If fetching succeed, then checking whether the idle time of the connection (since the last time it was placed in the connection pool) has exceeded `MaxIdleTimeout`, if yes, would close this connection and create a new connection
```

Comment:
If fetching succeed, then check whether the idle time of the connection (since the last time it was placed in the connection pool) has exceeded `MaxIdleTimeout`. If yes, this connection will be closed and a new connection will be created.